README.md

```diff
@@ -1 +1,59 @@
-# angular-project-setup-lab
+# Project Setup
+
+## Objectives
+
+- Download the latest version of Angular 1.x
+- Setup the directory structure for your Angular application
+ 
+## Instructions
+
+Go to the [Angular website](https://angularjs.org/) and download the latest version.
```

Comment:
We usually include libraries with the repo rather than having the student download it. I downloaded the minified version and it took me a second to realize that the index page was loading `js/angular.js`, not `js/angular.min.js`.